### PR TITLE
Add rnaQUAST and Sleuth

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

File: files/galaxy/tpv/tools.yml [hunk 1]
```diff
@@ -102,6 +102,20 @@ tools:
...
+    scheduling:
+      require:
+        - condor-tpv
+
+  tool_id=toolshed.g2.bx.psu.edu/repos/iuc/rnaquast/rna_quast/.*:
+    cores: 12
+    mem: 8
+    scheduling:
+      require:
+        - condor-tpv
```
Comment (kysrpex) [inline files/galaxy/tpv/tools.yml:117]: Why is `condor-tpv` a requirement?

**Review by kysrpex — CHANGES_REQUESTED** [review]
Should't we add the tools to [tpv-shared-database](https://github.com/galaxyproject/tpv-shared-database) instead?

**Comment by bgruening** [comment]
> Should't we add the tools to [tpv-shared-database](https://github.com/galaxyproject/tpv-shared-database) instead?

I think we need a SOP here.
Proposal: Users (Cristobal would be here a user) are adding stuff here, so we get it merged quickly and can test it without a large impact. Once a month the "admins" take all changes (could be maybe a CI?) and try to push them to the shared DB.

File: files/galaxy/tpv/tools.yml [hunk 2]
```diff
@@ -102,6 +102,20 @@ tools:
       prefer:
         - condor-tpv
 
+  toolshed.g2.bx.psu.edu/repos/iuc/sleuth/sleuth/.*:
+    cores: 4
+    mem: 16
+    scheduling:
+      require:
+        - condor-tpv
```
[the remark this answers is not in the document]
Comment (kysrpex) [reply files/galaxy/tpv/tools.yml:110]:
```suggestion
```

**Review by kysrpex — CHANGES_REQUESTED** [review]

File: files/galaxy/tpv/tools.yml [hunk 3]
```diff
@@ -102,6 +102,17 @@ tools:
...
+  toolshed.g2.bx.psu.edu/repos/iuc/sleuth/sleuth/.*:
+    cores: 4
+    mem: 16
+
+  toolshed.g2.bx.psu.edu/repos/iuc/rnaquast/rna_quast/.*:
+    cores: 12
+    mem: 8
+    scheduling:
+      require:
+        - condor-tpv
```
Comment (gallardoalba) [inline files/galaxy/tpv/tools.yml:114]:
```suggestion
```

**Review by kysrpex — APPROVED** [review]

**Pull request merged**